sideboard/lib/_cp.py

```diff
@@ -10,8 +10,9 @@
 import cherrypy
 
 import sideboard.lib
-from sideboard.lib import log, config
+from sideboard.lib import log, config, serializer
```

Comment:
The changes in this file all relate to #31 which MAGFest probably doesn't care much about, but was useful for my day job.